app/models/upvs/box.rb

```diff
@@ -20,13 +20,16 @@ def self.policy_class
     BoxPolicy
   end
 
-  validates_uniqueness_of :name, :short_name, :uri, scope: :tenant_id
-
   store_accessor :settings, :obo, prefix: true
 
+  validates_uniqueness_of :name, :short_name, :uri, scope: :tenant_id
+
   validate :validate_settings_obo
 
-  after_initialize :initialize_defaults, :if => :new_record?
+  normalizes :settings, with: -> (settings) {
+    settings["obo"] = settings.dig('obo').presence
```

Comment:
dig sa pouziva myslim len ked chces hlbsie zanorenie tu by mali stacit klasicke 

```suggestion
    settings["obo"] = settings["obo"].presence
```